helm/fiftyone-teams-app/values.yaml

```
@@ -31,6 +31,9 @@ apiSettings:
     repository: voxel51/fiftyone-teams-api
     # -- Image tag for teams-api. Defaults to the chart version.
     tag: ""
+  # Additional labels for the `teams-api` deployment. [Reference][labels-and-selectors].
```

To make the description appear in the rendered variable table, we need to instruct mkdocs with `# -- <COMMENT>`.  This will require running the pre-commits again to render the markdown through the template.


```suggestion
  # -- Additional labels for the `teams-api` deployment. [Reference][labels-and-selectors].
```